src/client/styles/Consents.ts

```diff
@@ -23,6 +23,12 @@ export const text = css`
   max-width: 640px;
 `;
 
+export const subText = css`
+  margin: 0;
+  ${textSans.small()}
+  max-width: 640px;
```

Comment:
I can't tell what the `max-width` is for here, in the usages I can see of `subText` the grid layout is what constrains the text width instead.